src/components/Common/CategoryRanking.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+const categories = [
+  "Fiction",
+  "Non-Fiction",
+  "Science Fiction",
+  "Fantasy",
+  "Mystery",
+  "Romance",
+  "Horror",
+  "Biography",
+  "History",
+  "Self-Help",
+];
```

Comment:
It's not ideal to define categories list in the front end. It's better get these list from the BE